gp-includes/things/original.php

```diff
@@ -80,13 +80,57 @@ public function by_project_id( $project_id ) {
 		return $this->many( "SELECT * FROM $this->table WHERE project_id= %d AND status = '+active'", $project_id );
 	}
 
-	public function count_by_project_id( $project_id ) {
-		if ( false !== ( $cached = wp_cache_get( $project_id, self::$count_cache_group ) ) ) {
-			return $cached;
+	/**
+	 * Retrieves the number of originals for a project.
+	 *
+	 * @since 1.0.0
+	 * @since 2.1.0 Added the `$type` parameter.
+	 *
+	 * @param int    $project_id The ID of a project.
+	 * @param string $type       The return type. 'total' for public and hidden counts, 'hidden'
+	 *                           for hidden count, 'public' for public count, 'all' for all three
+	 *                           values. Default 'total'.
+	 * @return object|int|null Object when `$type` is 'all', integer when `$type` is 'total',
+	 *                         'public' or 'hidden'. Otherwise null.
+	 */
+	public function count_by_project_id( $project_id, $type = 'total' ) {
+		global $wpdb;
+
```

Comment:
I would think the return should be 0, the function is effectively a question, 'how many originals of this type are there?', if it's an invalid type the answer is `0`.  It could be `null` if we needed to distinguish between a valid argument and an empty result, but I would think 'false' would be better in that case.

This is the same logic of the above `return null;`.